Userland/Libraries/LibGUI/Calendar.cpp

```diff
@@ -271,7 +275,7 @@ void Calendar::update_tiles(unsigned view_year, unsigned view_month)
             view_month = i + 1;
 
         auto first_day_of_current_month = Core::DateTime::create(view_year, view_month, 1);
-        unsigned start_of_month = (first_day_of_current_month.weekday() - m_first_day_of_week + 7) % 7;
+        unsigned start_of_month = (first_day_of_current_month.weekday() - to_underlying(m_first_day_of_week) + 7) % 7;
```

Comment:
This invocation to `to_underlying` also needed to be merged into the first commit. The idea is that commits should be atomic. As of now, the first commit wouldn't compile.